ArcGISRuntimeSDKQt_CppSamples/Search/FindAddress/FindAddress.cpp

```diff
@@ -78,7 +78,7 @@ void FindAddress::componentComplete()
 
   // create locator task and parameters
   //! [FindAddress create LocatorTask]
-  m_locatorTask = new LocatorTask(QUrl("https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer"), this);
+  m_locatorTask = new LocatorTask(QUrl("https://geocode-api.arcgis.com/arcgis/rest/services/World/GeocodeServer"), this);
```

Comment:
optional - might be worthwhile to add a comment indicating that api keys will be required